libbpf-tools/drsnoop.bpf.c

```diff
@@ -52,16 +53,14 @@ int handle__mm_vmscan_direct_reclaim_begin(u64 *ctx)
 }
 
 SEC("tp_btf/mm_vmscan_direct_reclaim_end")
-int handle__mm_vmscan_direct_reclaim_end(u64 *ctx)
+int BPF_PROG(direct_reclaim_end, unsigned long nr_reclaimed)
 {
 	u64 id = bpf_get_current_pid_tgid();
-	/* TP_PROTO(unsigned long nr_reclaimed) */
-	u64 nr_reclaimed = ctx[0];
 	struct piddata *piddatap;
 	struct event event = {};
 	u32 tgid = id >> 32;
 	u32 pid = id;
-	u64 delta_ns;
+	s64 delta;
```

Comment:
delta_ns was actually better, removes the guessing of time units

Reply:
Fixed, thanks! :)